src/ResourceGraph/ResourceGraph/ResourceGraph.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.1.0" />
```

Comment:
Please wait to the SDK general available, localfeed is not allowed for the time being.